app/models/story.rb

```diff
@@ -39,23 +39,36 @@ def self.paranoia_scope
   belongs_to :series, touch: true
   belongs_to :network
 
-  has_many :images, -> { where(parent_id: nil).order(:position) }, class_name: 'StoryImage', foreign_key: :piece_id
+  has_many :images,
+           -> { where(parent_id: nil).order(:position) },
+           class_name: 'StoryImage',
+           foreign_key: :piece_id,
+           dependent: :destroy
   has_many :audio_versions, -> { where(promos: false).includes(:audio_files) }, foreign_key: :piece_id
   has_many :audio_files, through: :audio_versions
-  has_many :producers
-  has_many :musical_works, -> { order(:position) }, foreign_key: :piece_id
-  has_many :topics, foreign_key: :piece_id
-  has_many :tones, foreign_key: :piece_id
-  has_many :formats, foreign_key: :piece_id
-  has_many :taggings, as: :taggable
+  has_many :producers, foreign_key: :piece_id, dependent: :destroy
+  has_many :musical_works,
+           -> { order(:position) },
+           foreign_key: :piece_id,
+           dependent: :destroy
+  has_many :topics, foreign_key: :piece_id, dependent: :destroy
+  has_many :tones, foreign_key: :piece_id, dependent: :destroy
+  has_many :formats, foreign_key: :piece_id, dependent: :destroy
+  has_many :taggings, as: :taggable, dependent: :destroy
   has_many :user_tags, through: :taggings
-  has_many :picks, foreign_key: 'playlistable_id'
+  has_many :picks, foreign_key: 'playlistable_id', dependent: :destroy
   has_many :playlists, through: :picks
   has_many :purchases, foreign_key: :purchased_id
-  has_many :distributions, class_name: StoryDistribution, foreign_key: :piece_id
-
-  has_one :promos, -> { where(promos: true) }, class_name: 'AudioVersion', foreign_key: :piece_id
-  has_one :license, foreign_key: :piece_id
+  has_many :distributions,
+           class_name: StoryDistribution,
```

Comment:
would vote we make this a string for consistency with the rest of the repo (plus [this](https://stackoverflow.com/questions/20390991/activerecord-pass-class-instead-of-string-to-class-name-when-defining-associati) says so) 
`class_name: 'StoryDistribution',`

(officially nitpickier than hound! 🐶 )